robosherlock/src/queryanswering/CMakeLists.txt

```diff
@@ -14,3 +15,12 @@ add_library(rs_prologQueries SHARED src/PrologCPPQueries.cpp)
 target_link_libraries(rs_prologQueries ${robosherlock_LIBRARIES} 
     ${SWIPL_LIBRARIES} ${catkin_LIBRARIES})
 add_dependencies(rs_prologQueries ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+
+
+rs_add_executable(RSSimpleQueryActionClient src/RSSimpleQueryActionClient.cpp)
+target_link_libraries(RSSimpleQueryActionClient rs_queryanswering ${catkin_LIBRARIES})
+
+rs_add_executable(RSSimpleQueryActionServer src/RSSimpleQueryActionServer.cpp)
+target_link_libraries(RSSimpleQueryActionServer rs_queryanswering ${catkin_LIBRARIES})
+
```

Comment:
are these executables necessary? They look more like executables for testing. Delete please.

Reply:
Yes, just for testing. I deleted the **Server executable** and left the client since it appears to be necessary for testing